### ISSUE TITLE
Grand Boss Antharas

### ISSUE THREAD

**Issue by maneco2** [L2J/L2J_DataPack#193]
Npc ID: 29068
It has a skill he must use only the script, it can not contain 2 times: (4112, 1); // Ordinary Attack
This skill can not be at npc
In L2J Antharas is giving is always skill in sequence as many times.
I'm updating the droplist all forgottens retail, I also see it.
Remove this skill in XML: <skill id="4112" level="1" />  (already have in script)

**Comment by Zoey76** [comment]
No, the skill should not be removed from the skill list, needs to be handled in other way.

**Comment by maneco2** [comment]
Zoey the ability contains in the script, Antharas.java
